### DISCUSSION TITLE
[Question] Custom import robot arm + claw, claw can not pick up objects

### DISCUSSION THREAD

**Discussion by qiangl9618** [isaac-sim/IsaacLab#1151] [Q&A]
### Question

I used urdf-usd in isaaclab to import my own mechanical arm + claw, but when I was clamping objects, I found that once the clamping objects collided, it would cause joint rotation. Finally, after debugging, I thought it was the problem of colliding bodies between the claws, but I didn't know how to deal with it.
[nvidia夹爪碰撞问题.webm](https://github.com/user-attachments/assets/82e0433c-8eee-4210-8547-0b7fe4cfae62)

[nvidia夹爪-抓取.webm](https://github.com/user-attachments/assets/c6179e29-aef3-4e7f-bc8a-1844f8f5b7e2)

**Comment by kellyguo11** [comment]
Hi there, it may be helpful to tune the damping or stiffness of the joints for the robot

**Comment by ncast-dev** [comment]
## FACT-053 — FV2-048 Migration script v1 → v2 DB ✅

**Commit:** 7deb42d
**PR:** https://github.com/ncast-dev/Agent-hub/pull/2243

### Livré
- `scripts/migrate_v1_to_v2.py` : script standalone v1 SQLite → v2 PostgreSQL
- `tests/test_FACT-053.py` : 11 tests GREEN (SQLite in-memory, zéro API externe)

### Ce que fait le script
1. Crée le tenant `nicolas` (idempotent)
2. Construit un slug map depuis les projects v1 (`my-project` etc.)
3. Migre les sprints v1 → `sprints` v2 avec `tenant_id` + `project_slug`
4. Migre les stories v1 → `orch_stories` v2 (liées au sprint + tenant)
5. Skip les orphelines (stories sans sprint_id)
6. Support `--dry-run` : lecture seule
7. Idempotent : relancer = skip des rows déjà migrées

### Usage
```
python scripts/migrate_v1_to_v2.py \n  --v1-db /path/to/factory-cockpit/factory.db \n  --v2-url postgresql+psycopg2://user:pass@100.119.27.64:5432/factory_v2 \n  --tenant-slug nicolas \n  --tenant-name "Nicolas Castoriadis"
```
